backend/StreamServerApp/migrations/0012_video_subtitle.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.3 on 2019-08-25 12:00
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('StreamServerApp', '0011_video_video_folder'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='video',
+            name='subtitle',
```

Comment:
Maybe would make more sense to be more explicit and name this field `subtitle_ulr`.